### ISSUE TITLE
Corona-Warn-App for Apple-Watch

### ISSUE THREAD

**Issue by ststoessel** [corona-warn-app/cwa-wishlist#12]
Please consider an additional Corona-Warn-App for the Apple-Watch.

**Comment by cfritzsche** [comment]
@tkowark As you accepted this idea into the wishlist, it would make sense to reopen the issue. It was closed two days before you moved it.

**Comment by GisoSchroederSAP** [comment]
Tobias Dietrich <tobias-dietrich@t-online.de>
Fri 9/18/2020 10:02 PM
To: SAP Corona Warn-App Open Source Team
Guten Tag,

Ich möchte vorschlagen, die Corona Warn App auch für Wearables (zB Apple Watch) anzubieten, da manche Berufe mit viel Kontakt zu anderen Personen gleichzeitig das mitführen von Mobiltelefonen nicht gestatten, das tragen von Smartwatches aber schon. Wenn man nun did Zufallstores auch mit der Apple Watch oder ähnlichen Geräten sammeln könnte, würde dies zu genaueren Ergebnissen führen, da dann die falschen „Positiv“Ergebnisse verringert würden, wenn man nur deswegen keinen Kontakt ermittelt bekommt, weil das Handy nicht am Körper war.


Mit freundlichen Grüßen,

Tobias Dietrich

**Comment by Ein-Tim** [comment]
Related to: #91

**Comment by iMonZ** [comment]
> Please consider an additional Corona-Warn-App for the Apple-Watch.
> 
> Internal Tracking ID: EXPOSUREAPP-2180

Supporting the Apple wallet would makes much more sense than creating a whole new app for the Apple Watch which is kinda unnecessary.

**Comment by jucktnich** [comment]
@GisoSchroederSAP if Apple would integrate the ENF into the Apple Watch, it would probably be synced with the iPhone, so there would be no reason for an extra app.

**Comment by ShadNex** [comment]
I think it would be a really useful addition to have the Vaccination Certificate on the Apple Watch. That way if you need to present it to someone for scanning and verifying, it's much quicker than on the phone (maybe even support for a watch face complication, so it's really just one tab away?). It basically just has to show the QR Code which is mirrored from the phone (shouldn't be too complicated?).

**Comment by jucktnich** [comment]
@ShadNex I think the wallet integration sufficient for ur usecase

**Comment by ShadNex** [comment]
@jucktnich oh sorry, I didn't know about the Wallet Integration. Where can I find more info about it? Is this already active?

**Comment by jucktnich** [comment]
Nope, it's discussed, sorry if my answer wasn't clear. But if the wallet thing would be integrated, what I hope, this wouldn't be needed. The development process for a wallet integration would be much faster.

**Comment by iMonZ** [comment]
The wallet integration would be the best solution!
For vaccination and negative tests
Here:
https://developer.apple.com/wallet/
(Actually it’s pretty easy and should be done in less than a few days)
The problem is for the approval of this feature.
Currently nobody at the team cares about this function which is pretty sad

**Comment by ststoessel** [comment]
Somehow the original intention of this feature request got lost.

One year ago my intention was that a watch device will always be with the owner. A mobile phone probably not.

So a Watch App should be primarily for supporting its original focus on monitoring contacts.

Making the digital vaccine certificate available on a watch is something completely different and to me of minor importance.


> Am 18.06.2021 um 11:03 schrieb Goldgamer ***@***.***>:
> 
> The wallet integration would be the best solution

**Comment by Ein-Tim** [comment]
@ststoessel 

For this it first has to be clarified if the ENF uses the Apple Watch (see https://github.com/corona-warn-app/cwa-app-ios/issues/2822).

**Comment by iMonZ** [comment]
> Somehow the original intention of this feature request got lost. One year ago my intention was that a watch device will always be with the owner. A mobile phone probably not. So a Watch App should be primarily for supporting its original focus on monitoring contacts. Making the digital vaccine certificate available on a watch is something completely different and to me of minor importance.
> […](#)

So the Apple Watch is every time with me but my iPhone stays a lot of the time at home because I don’t need it. It’s just an distraction for me that’s why the watch certificate support would be the best

**Comment by ststoessel** [comment]
The original feature request was hijacked.
The request for a Watch App had a total different focus.

The certificate request is a newly requested feature and should get a new different Github request.

**Comment by heinezen** [comment]
@ststoessel 

Can you amend your original post in this thread to mention what your original feature request was meant to be? I assume you wanted a standalone release of the app on Apple Watch.

----
Corona-Warn-App Open Source Team

**Comment by ststoessel** [comment]
I made the intention of the Watch-App clearer in the starting post.

**Comment by 10Meisterbaelle** [comment]
It should also be able to show your Covid Passes

**Comment by Ein-Tim** [comment]
@10Meisterbaelle please see https://github.com/corona-warn-app/cwa-wishlist/issues/12#issuecomment-864000856 and #563 for this.

**Comment by GisoSchroederSAP** [comment]
There are multiple topics mixed here, so please let me provide brief statements - and yes, then you may decide for seperate issues in GitHub: 

1. Support for wearable (including Apple or other smart watches: Currently no plan at all because of several technical and logistical reasons, sorry. 
2. Support of Apple's wallet (or any other wallet function): As long as nobody can ensure that the wallet does *not* bring certificates close to other content carrying personal and/or location details we cannot/must not/are not permitted to use those wallets.
3. Presenting (vaccination) certificates within the Apple watch: Here we move away from the original idea of a Warn App, right? Just to easily present the certificate. I hardly believe we can motivate this request for the customer - even with the reasoning, a smartphone is not allowed in certain situations, but a smart watch is allowed: The same reasoning we will receive for the "paper version" of the certificate. So, if you leave the smartphone home, just grab the piece of paper with the QR code. 

I totally understand, my statements will lead to discussions and further remarks. Please do so! I'll be off for a couple of days and will pick up the topics afterwards.

**Comment by TobeyWins** [comment]
@GisoSchroederSAP To your 3. point: I really like using the CWA but getting the Certificate requires too many clicks to be convenient. An App to shwo the certificate for the watch would just be great. The tracking and full view could stay on the phone but the certificate itself would be very handy on the watch.
I'm not willing to use paper verison but it would just be so convenient if an apple watch integration is possible.

**Comment by jucktnich** [comment]
@TobeyWins with the Austrian app you can add it to the wallet and therefor to the watch. But for cheaper devices it's a hustle to scan the cert from the small screen.
BTW I'm supporting your idea

**Comment by larswmh** [comment]
Closing this issue as there is currently no Corona-Warn-App for Apple Watch planned.
